sql/moz-fx-data-shared-prod/sumo_ga/dataset_metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: SUMO.org Google Analytics
```

Comment:
SUMO's domain is support.mozilla.org, so SUMO or support.mozilla.org are both correct but sumo.org is wrong.

Reply:
Good catch, updated this! thanks!